mobile/src/core/theme/colors.ts

```diff
@@ -1,13 +1,83 @@
-interface IColors {
-  white: string
-  grey: string
-  black: string
-  blue: string
-}
-
-export const colors: IColors = {
-  white: '#FFFFFF',
-  grey: '#F2F2F2',
-  black: '#262626',
-  blue: '#00579D'
+export enum Colors {
+  transparent = 'transparent',
+  white = '#FFFFFF',
+  black = '#262626',
+  greyLight = '#F2F2F2',
+  grey = '#CDCDCD',
+  greyDark = '#595959',
+  blue = '#00579D',
+  blueDark = '#002341',
+  pink = '#E91E63',
+  red = '#F44336'
+}
+
+export interface ThemeColors {
+  color: Colors;
```

Comment:
??? seems to be a preparation for future theming

Reply:
In my mind it's more like "component on dark/light/somewhat place", but it's still a concept that can (and probably will) be changed